src/Esfa.Vacancy.Api.Types/SearchApprenticeshipParameters.cs

```diff
@@ -24,5 +24,10 @@ public class SearchApprenticeshipParameters
         /// Page number is optional and if not specified, by default the first page is served.
         /// </summary>
         public int PageNumber { get; set; } = 1;
+
+        /// <summary>
+        /// Vacancies posted in last number of days from today.
+        /// </summary>
+        public int PostedInDays { get; set; }
```

Comment:
I know we don't like long names but I would suggest PostedInLastNoOfDays to indicate past whereas the current name is ambiguous and could infer future.

Reply:
fixed